docs/user/guides/pull-requests.rst

```diff
@@ -53,15 +53,14 @@ Privacy levels work the same way as :ref:`normal versions <versions:Version stat
 Limitations
 -----------
 
-- Only available for **GitHub** and **GitLab** currently. Bitbucket is not yet supported.
+- Pull requests are only available for **GitHub** and **GitLab** currently. Bitbucket is not yet supported.
 - To enable this feature, your Read the Docs account needs to be connected to an
   account with your Git provider.
 - Builds from pull requests have the same memory and time limitations
   :doc:`as regular builds </builds>`.
-- Additional formats like PDF and EPUB aren't built, to reduce build time.
-- Search queries will default to the default experience for your tool.
-  This is a feature we plan to add,
-  but don't want to overwhelm our search indexes used in production.
+- Additional formats like PDF aren't built in order to reduce build time.
+- Search queries will default to the default experience for your tool,
+  and won't use Read the Docs search.
```

Comment:
This isn't super clear I think.
Also, does it use the PR index or `main`?

Reply:
This can probably be updated, since we aren't injecting tool-specific JS anymore :tada: